src/v2i-hub/CDASimAdapter/scripts/send_timestep_udp.py

```diff
@@ -9,13 +9,16 @@
 # host and port. To Test the Time Sync functionality of the CDASimAdapter
 # set port to the value of the TIME_SYNC_PORT environment variable.
 #
-# TODO Move this script into a more permanent location
+# count_name used to populate the seq field in the time sync message
 count_num = 0
-
+# sim_time (ms) used to populate the timestep field in the time sync message
+sim_time =0
+# sleep_dur (s) used to control the rate at which time sync messages are sent
+sleep_dur = 0.1
```

Comment:
nit: Its usually better for readability to include units in variable names, suggest renaming this variable instead of just adding in comment